solr/server/etc/jetty.xml

```diff
@@ -178,7 +178,7 @@
                  </New>
                </Item>
                <Item>
-                 <New id="InstrumentedHandler" class="com.codahale.metrics.jetty9.InstrumentedHandler">
+                 <New id="InstrumentedHandler" class="com.codahale.metrics.jetty10.InstrumentedHandler">
```

Comment:
Ugh @epugh  found that this is broken - it needs to be `io.dropwizard.metrics.jetty10.InstrumentedHandler`

Reply:
Fixed in 1c9ea3ad9ac98cc133163f3e36c643ca567009a6